### PR TITLE
Update old hard-coded link to current one

### PR DESCRIPTION
**Pull request by insukcho**
Founded this from [here](https://www.elastic.co/guide/en/cloud/current/snapshot-restore.html)

### CODE REVIEW

File: html/en/cloud/saas-release/snapshot-restore.html [hunk 1]
```diff
@@ -1177,4 +1177,4 @@ <h3><a href="/about">About ></a></h3>
 
             </body>
         
-</html>
```
Comment (insukcho) [inline html/en/cloud/saas-release/snapshot-restore.html]: I'm not sure why this line has been changed. Looks like getting rid of special char when edit file. I tried to edit in github UI and also vi editor on mac terminal.

**Comment by nrichers** [comment]
@insukcho Thank you for your PR. The Elastic docs are sourced in Asciidoc and the HTML output you are trying to update gets overwritten regularly. Changes should be made only to the Asciidoc source in https://github.com/elastic/cloud/blob/saas-release/docs/saas/snapshot-restore.asciidoc. I will close this PR, but if you want some help making updates, just ping me on Slack?

**Comment by insukcho** [comment]
@nrichers thanks for fixing this link to current one. 
Just curious, how can I find :ref: variable in asciidoc which you point out in previous comment?

**Comment by insukcho** [comment]
Have founded [PR](https://github.com/elastic/cloud/pull/12716/). 
Thanks again!

**Pull request closed without merging**